src/txn/txn_recover.c

```diff
@@ -215,16 +223,16 @@ __txn_system_apply(WT_RECOVERY *r, WT_LSN *lsnp, const uint8_t **pp, const uint8
 /*
  * Helper to a cursor if this operation is to be applied during recovery.
  */
-#define GET_RECOVERY_CURSOR(session, r, lsnp, fileid, cp)                  \
-    ret = __recovery_cursor(session, r, lsnp, fileid, false, cp);          \
-    __wt_verbose_debug2(session, WT_VERB_RECOVERY,                         \
-      "%s op %" PRIu32 " to file %" PRIu32 " at LSN %" PRIu32 "/%" PRIu32, \
-      ret != 0         ? "Error" :                                         \
-        cursor == NULL ? "Skipping" :                                      \
-                         "Applying",                                       \
-      optype, fileid, (lsnp)->l.file, __wt_lsn_offset(lsnp));              \
-    WT_ERR(ret);                                                           \
-    if (cursor == NULL)                                                    \
+#define GET_RECOVERY_CURSOR(session, r, lsnp, fileid, cp)         \
```

Comment:
Note for reviewers: This macro is used exclusively in `__txn_op_apply`. The WT_ITEM is declared in that function.